### PR TITLE
additional checks for successful result

### PR DESCRIPTION
**Pull request by corymosiman12**

### CODE REVIEW

File: lib/openstudio/extension/runner.rb [hunk 1]
```diff
@@ -656,8 +656,15 @@ def run_osw(in_osw, run_dir)
...
-        # DLM: this does not always return false for failed CLI runs, consider checking for failed.job file as backup test
+        # Additional checks for failed CLI
+        if File.exist?(File.join(run_dir, 'failed.job'))
+          result = false
+        end
 
+        if !File.exist?(File.join(run_dir, 'finished.job'))
+          result = false
+        end
+
```
Comment (nllong) [inline lib/openstudio/extension/runner.rb]: what happens if both a `failed.job` and `finished.job` exist? The code here will still return false.

**Review by nllong — CHANGES_REQUESTED** [review]
i think this needs to be based off of a 0.2.x branch -- like this SHA:https://github.com/NREL/openstudio-extension-gem/commit/f228d8fa96d70522d86c798b872afd33c13d5ed1

File: lib/openstudio/extension/runner.rb (hunk 1 above)
Comment (corymosiman12) [reply lib/openstudio/extension/runner.rb]: yeah but that makes sense to me.

**Comment by corymosiman12** [comment]
Deleting

File: lib/openstudio/extension/runner.rb (hunk 1 above)
Comment (nllong) [reply lib/openstudio/extension/runner.rb]: okay. looks good. i approve 👍  😜

**Pull request closed without merging**